Makefile

```diff
@@ -1,6 +1,3 @@
-.DEFAULT_GOAL := help
```

Comment:
`.DEFAULT_GOAL` should stay. `help` is the first target, so `make` defaults to using it, but the presence of this serves as an explicit callout that this is the intention.

Reply:
✅